cst438_gradebook/src/main/java/com/cst438/domain/EnrollmentDTO.java

```diff
@@ -1,17 +1,25 @@
 package com.cst438.domain;
 
 public class EnrollmentDTO {
-	public int id;
-	public String studentEmail;
-	public String studentName;
-	public int course_id;
-	
-	public EnrollmentDTO() { } ;
-	
-	public EnrollmentDTO(String studentEmail, String studentName, int course_id) {
-		this.id = 0;
-		this.studentEmail=studentEmail;
-		this.studentName=studentName;
-		this.course_id = course_id;
-	}
+    public int id;
+    public String studentEmail;
+    public String studentName;
+    public int course_id;
+
+    public EnrollmentDTO() {
+    }
+
+    public EnrollmentDTO(Enrollment enrollment) {
+        this.id = enrollment.getId();
```

Comment:
This was added to create another public EnrollmentDTO. For strings studentEmail and studentName. In addition, this would help set up course_id.